MANIFEST.in

```diff
@@ -1,11 +1,11 @@
 recursive-include dask *.py
-recursive-include docs *.rst
+recursive-include docs/source *
+include docs/Makefile docs/make.bat
```

Comment:
These files aren't part of the nominal set of source files required to generate the Sphinx documentation. Should they be included, it would be the subject of a different PR.